algorithms/CPlusPlus/Linked-Lists/sadkjasdkajsdk.cpp

```diff
@@ -0,0 +1,9 @@
+#include<bits/stdc++.h>
+#include <array>
+using namespace std;
+int main(){
+	std:: array<int, 5> myarray={1,2,3,4,5};
+	for(int i=0;i<array.size();i++){
+		cout<<myarray[i];
+	}
+}
```

Comment:
I don't this file is relevant to the commit.
Please remove it

Reply:
This was done by mistake. Updated the changes that you have suggested and thanks for pointing out my mistakes